test/unit/test-cid.js

```diff
@@ -823,6 +832,19 @@ describes.realWin('cid', {amp: true}, (env) => {
     });
     cid = cidServiceForDocForTesting(ampdoc);
     env.sandbox.stub(cid.cacheCidApi_, 'isSupported').returns(false);
+    storageSetStub = stubServiceForDoc(
```

Comment:
You can refer to how we stub the storage service in amp-consent test
https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/0.1/test/test-amp-consent.js#L92

I think it'll be helpful.